packages/client/components/MeetingHelp/ReflectHelpMenu.tsx

```diff
@@ -19,10 +35,12 @@ const ReflectHelpMenu = forwardRef((_props: Props, ref: any) => {
       <HelpMenuCopy>
         As a group, reflect on how work’s going for a specific project or timeframe.
       </HelpMenuCopy>
-      <HelpMenuCopy>
-        During this phase nobody can see your reflections. After this phase reflections will be
-        visible, but remain anonymous.
-      </HelpMenuCopy>
+      {!disableAnonymity && (
+        <HelpMenuCopy>
+          During this phase nobody can see your reflections. After this phase reflections will be
+          visible, but remain anonymous.
```

Comment:
-1 I think we should keep the paragraph and only put "but remain anonymous" behind the `disableAnonymity` check.

Reply:
Updated the code.